src/unity/python/turicreate/connect/main.py

```diff
@@ -16,15 +16,12 @@
 import decorator
 import logging
 
-""" The module level logger object """
-__LOGGER__ = logging.getLogger(__name__)
```

Comment:
I don't think we should delete this one.  It could be helpful for interactive debugging.